test/e2e/agent_test.go

```diff
@@ -2,18 +2,18 @@ package agent_test
 
 import (
 	"fmt"
-	"os"
-	"path/filepath"
 	"strings"
 	"testing"
 
-	"github.com/flightctl/flightctl/test/harness/e2e/vm"
-	"github.com/google/uuid"
+	"github.com/flightctl/flightctl/internal/api/client"
+	"github.com/flightctl/flightctl/test/harness/e2e"
+	testutil "github.com/flightctl/flightctl/test/util"
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
+	"github.com/sirupsen/logrus"
 )
 
-const TIMEOUT = "30s"
+const TIMEOUT = "1m"
```

Comment:
just curious why we need to double the timeout?

Reply:
The TPM test is more complex and takes longer. (Waiting for enrollment, approval, waiting for device report to check again the TPM details)... 

We could bump that case , I am trying to remember and may be some of the other tests also timed out. We could be more specific per test I guess.